db/schema.rb

```diff
@@ -33,6 +33,39 @@
   add_index "authentications", ["uid", "provider"], :name => "index_authentications_on_uid_and_provider", :unique => true
   add_index "authentications", ["user_id", "provider"], :name => "index_authentications_on_user_id_and_provider", :unique => true
 
+  create_table "c_rater_feedback_items", :force => true do |t|
```

Comment:
Small update on that. In fact I'm implementing the first option finally.

The problem is about displaying old feedback when user visits page again. I was thinking that for multiple choice answer we could always get feedback dynamically. But... it creates problems when user first selects answer "A", clicks "Submit", then changes answer to "B" and reloads the page.

I assume that we always want to display feedback that was explicitly created by submit action (so feedback on "A") or nothing if answer has been changed in the meantime. I bet we don't want "silently" update feedback to match "B" answer. So we need explicit storage for feedback. Using linking model for that described in 2 would be messier.